packages/cli/src/info/env_info_provider.ts

```diff
@@ -16,15 +16,33 @@ export class EnvironmentInfoProvider {
         System: ['OS', 'CPU', 'Memory', 'Shell'],
         Binaries: ['Node', 'Yarn', 'npm', 'pnpm'],
         npmPackages: [
+          '@aws-amplify/auth-construct',
```

Comment:
nit: what if we create new packages? Should we add packages with `@aws-amplify` prefix automatically?

Reply:
I thought about this but decided to punt on it for now. Reason being we are unlikely to add packages frequently and needing versions for debugging will be a forcing function for adding any missing packages to this list. There's also no functional impact to a missing package here, just a bit more back and forth on issue repro.

Reply:
I also trie to add a glob pattern `@aws-amplify/*` but the `envinfo` package does not support that.